dotnet.cmd

```diff
@@ -11,14 +17,19 @@ set DOTNET_MULTILEVEL_LOOKUP=0
 :: Disable first run since we do not need all ASP.NET packages restored.
 set DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
 
-:: Install dotnet
-call %~dp0\init-dotnet.cmd
-if NOT [%ERRORLEVEL%]==[0] (
+echo Installing dotnet using Arcade...
+set PS_DOTNET_INSTALL_SCRIPT=". %__RepoRootDir%eng\configure-toolset.ps1; . %__RepoRootDir%eng\common\tools.ps1; InitializeDotNetCli($true)"
```

Comment:
You don't need the dot include of configure-toolset.ps1: `. %__RepoRootDir%eng\configure-toolset.ps1;`. Tools.ps1 does that itself.

Reply:
Deleted in 3rd commit.